### ISSUE TITLE
Blank dashboard view after editing a Singlestat panel with query that returns multiple series

### ISSUE THREAD

**Issue by shaunmansell** [grafana/grafana#18753]
**What happened**:

1) Added a singlestat panel to a new dashboard and set the query `up{}`

2) Click away from the query text field. Query executes and the singlestat show `N/A` as expected, but no error is show in the UI that the query failed.

In the browser developer console we see the following reported twice:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/21325844/63832704-b6d1a680-c9b4-11e9-8764-63bdce7772d5.png">

3) Click the back arrow to go back to the main dashboard view. We still see the single stat planel showing `N/A`

<img width="1594" alt="image" src="https://user-images.githubusercontent.com/21325844/63833122-948c5880-c9b5-11e9-9209-e62aac7eb818.png">

4) Attempt to edit the single stat. The entire dashboard goes blank. The top context bar is gone along with all panels

<img width="1599" alt="image" src="https://user-images.githubusercontent.com/21325844/63833161-b8e83500-c9b5-11e9-9cde-5837d358597a.png">

In the browser developer console we see the following reported twice:

<img width="513" alt="image" src="https://user-images.githubusercontent.com/21325844/63832990-4b3c0900-c9b5-11e9-821f-5e666243a428.png">

There's no way that I've found to get the dashboard content back without refreshing the whole page. Any unsaved changes are lost as a result.

I've also found the same thing happens (blank dashboard view) if you switch a Graph panel with multiple series being returned by a query to a Singlestat.

**What you expected to happen**:

On executing the query and encountering multiple series for a singlestat, I'd expect Grafana to show the execution error on the top left of the panel reporting the underlying failure reason.

**How to reproduce it (as minimally and precisely as possible)**:

Follow the outline in `What happened`

**Anything else we need to know?**:

**Environment**:
- Grafana version: 6.3.3
- Data source type & version: Prometheus v2.9.1
- OS Grafana is installed on: Official Grafana Docker Container
- User OS & Browser: MacOS and Firefox 68.0.1
- Grafana plugins:
- Others:

**Comment by dprokop** [comment]
Same problem occurs when switching between SingleStat and SingleStat(react) when multiple queries added to the panel

**Comment by dprokop** [comment]
Duplicate of #17778

**Comment by shaunmansell** [comment]
I'm hitting this issue with a singlestat that only has one query, but the query results contain multiple series. 

As expected, the issue does not occur when a single on the panel query returns one series.

I understand that trying to show multiple series on a singlestat is invalid and the reason why. The main complication is that when you hit this issue you can't actually edit the queries (regardless of whether you have two queries, or one returning multiple series), since there's no way to get back to the query editing section of the singlestat panel.

It's not uncommon for me to switch visual types first, and then fix queries that may be invalid for the type I've switched to. This flow worked fine in v6.1, but now I have to remember to check and fix a query if it's not going to work on the new visual type. If you forget, then you're going to get stuck with a broken dashboard.

This is the singlestat react plugin right? - https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/singlestat2/plugin.json

How do I enable access to this in Grafana?

**Comment by dprokop** [comment]
> How do I enable access to this in Grafana?

In Grafana settings file add:

```
[panels]
enable_alpha = false
```

But please note that the alpha plugins are not feature complete yet.

**Comment by shaunmansell** [comment]
Thanks! I've tried out the react singlestat and do not encounter the same blank dashboard following the same steps I outlined in the issue description.

Interestingly the react singlestat shows the value of each series returned by the query instead of reporting a multi series error. That may be expected with the stage of development it's at.

<img width="659" alt="image" src="https://user-images.githubusercontent.com/21325844/63922344-db4a8300-ca87-11e9-8cef-e87872fb802a.png">

If I take that react panel setup and switch to the angular panel, I get a multiple series error as we'd expect:
<img width="331" alt="image" src="https://user-images.githubusercontent.com/21325844/63922524-32505800-ca88-11e9-96bd-6d6f84c66938.png">

but as soon as I click the `Queries` button to update the query, or hit the `Go back` button to return to the dashboard view I encounter the empty page.
